jquery.marquee.js

```diff
@@ -121,8 +121,10 @@
                 }
             });
 
-            // since speed option is changed to duration, to support speed for those who are already using it
-            o.duration = o.speed || o.duration;
+            // Reintroduce speed as an option. It calculates duration as a factor of the container width
+            if (o.speed) {
+                o.duration = o.speed * parseInt($this.width(), 10);
```

Comment:
This calculation is incorrect.
Science says: ``Velocity = Distance / Time`` or ``Time = Distance / Velocity``

Since:
``duration = Time`` milliseconds
``speed = Velocity`` ???
``width = Distance`` pixels

Your equation says: ``Time = Velocity * Distance`` or ``Time = Distance * Velocity``

This line should read: ``o.duration = parseInt($this.width(), 10) / o.speed``
Where: ``o.speed`` = _pixels per millisecond_

Or: ``o.duration = parseInt($this.width(), 10) / o.speed * 1000``
Where: ``o.speed`` = _pixels per second_